Makefile

```diff
@@ -157,12 +157,24 @@ fmt: ## Check formatting of go sources.
 	@$(MAKE) go.init
 	@$(MAKE) go.fmt
 
+
+.PHONY: markdownlint
+# the markdownlint  target requires
+# markdownlint-cli2 to be available locally in the PATH
+markdownlint: ## Check formatting of documentation sources
+	@if ! type markdownlint-cli2 ; then \
+	    echo "please install markdownlint-cli2 to use 'make markdownlint'" ; \
```

Comment:
Is there a recommended way to install? Can we include that in the message, or else document in the dev guide?

Reply:
@travisn  wrote:
> Is there a recommended way to install?

Not quite sure. I figured out ways to install.

>  Can we include that in the message, or else document in the dev guide?


Mentioning in the dev guide makes sense if we also mention `make markdownlint`there.

I can prepare such an addition to this PR.




 Can we include that in the message, or else document in the dev guide?

Reply:
@travisn  I have added a commit to expand the dev guide with instructions for running various linters locally. PTAL.

Reply:
@travisn I  added a single commit updating the dev guide.

This commit adds both a general update about locally running linters and instructions specific to markdownlint.

Now I am thinking if it would be better to split this commit into two according to these two parts and possibly break the general part out into a separate prerequisite PR. Thoughts?